packages/tasit-identity-contract/src/GnosisSafe.test.js

```diff
@@ -60,138 +61,211 @@ describe("GnosisSafe", () => {
     expect(provider._events).to.be.empty;
   });
 
-  describe("test cases that needs ETH deposit to the wallet", async () => {
+  describe("test cases that need ETHs deposit to the contract-based account", async () => {
```

Comment:
```suggestion
  describe("test cases that need ETH deposit to the contract-based account", async () => {
```

Reply:
Even though ETH is countable so it's valid to say "ethers" for a certain integer number of ETH, it's almost infinitely divisible, so I treat it as a continuous thing like water that doesn't become plural usually